javascript/index.js

```diff
@@ -0,0 +1,127 @@
+// email input
+const inputEmail = document.querySelector(".sign-input");
+const inputBoxEmail = document.querySelector(".sign-input-box");
+// password input
+const inputPassword = document.querySelector(".sign-input#password");
+const inputBoxPassword = document.querySelector(".sign-input-box.sign-password");
+// login button
+const loginButton = document.querySelector(".cta");
+// eye button & img
+const eyeButton = document.querySelector(".eye-button");
+const eyeImg = document.querySelector("#eye-id");
+
+function isValidEmail(emailAdress) {
+    let exptext = /^[A-Za-z0-9_\.\-]+@[A-Za-z0-9\-]+\.[A-Za-z0-9\-]+/;
+    return exptext.test(emailAdress);
+}
+// focus-out email
+function eventFocusOutEmail() {
+    // no email value
+    if(inputEmail.value === "") {
+        if(inputBoxEmail.lastElementChild === inputEmail) {
+            const nothingInput = document.createElement("p");
+            nothingInput.textContent = "이메일을 입력해주세요";
+            inputBoxEmail.append(nothingInput);
+
+            inputEmail.classList.add('error-border');
+            nothingInput.classList.add('error-text');
+        } else {
+            inputBoxEmail.lastElementChild.textContent = "이메일을 입력해주세요";
+        }
+    }
+    // not email form
+    else if(!isValidEmail(inputEmail.value)) {
+        if(inputBoxEmail.lastElementChild === inputEmail) {
+            const invalidInput = document.createElement("p");
+            invalidInput.textContent = "올바른 이메일 주소가 아닙니다.";
+            inputBoxEmail.append(invalidInput);
+
+            inputEmail.classList.add('error-border');
+            invalidInput.classList.add('error-text');
+        } else {
+            inputBoxEmail.lastElementChild.textContent = "올바른 이메일 주소가 아닙니다.";
```

Comment:
c: 에러메시지 추가, 변경, 삭제를 함수로 분리하면 email, 비밀번호의 에러 ui 대응 시 재사용 가능해 보입니다.

```suggestion
            const showErrorMessage = ({
              element,
              message,
            }) => {
                /**
                 * 에러 UI 추가 로직
                 */
              }
```